### PR TITLE
Document polling interval units

### PR DESCRIPTION
**Pull request by 9999years**
`Int` is ambiguous.

### CODE REVIEW

**Comment by thomasjm** [comment]
The three fields are exported:

https://github.com/haskell-fswatch/hfsnotify/blob/7e0802a8e2599c15baf4df0683d1f4c7072370c1/src/System/FSNotify.hs#L54-L57

I'll take the change documenting `watchModePollInterval` though.

**Comment by 9999years** [comment]
Thanks, let me rebase.

File: src/System/FSNotify/Types.hs [hunk 1]
```diff
@@ -55,7 +55,10 @@ type EventAndActionChannel = Chan (Event, Action)
 
 -- | Method of watching for changes.
 data WatchMode =
-  WatchModePoll { watchModePollInterval :: Int }
+  WatchModePoll
+    { watchModePollInterval :: Int
+      -- ^ Polling interval in microseconds.
+    }
```
Comment (thomasjm) [inline src/System/FSNotify/Types.hs:61]:
```suggestion
  WatchModePoll { 
    watchModePollInterval :: Int
    -- ^ Polling interval in microseconds.
  }
```

**Review by thomasjm — CHANGES_REQUESTED** [review]
Formatting nit

**Pull request merged**